mods/walls/init.lua

```diff
@@ -35,12 +39,12 @@ walls.register = function(wall_name, wall_desc, wall_texture, wall_mat, wall_sou
 
 end
 
-walls.register("walls:cobble", "Cobblestone Wall", "default_cobble.png",
+walls.register("walls:cobble", "Cobblestone Wall", {"default_cobble.png"},
 		"default:cobble", default.node_sound_stone_defaults())
 
-walls.register("walls:mossycobble", "Mossy Cobblestone Wall", "default_mossycobble.png",
+walls.register("walls:mossycobble", "Mossy Cobblestone Wall", {"default_mossycobble.png"},
 		"default:mossycobble", default.node_sound_stone_defaults())
 
-walls.register("walls:desertcobble", "Desert Cobblestone Wall", "default_desert_cobble.png",
+walls.register("walls:desertcobble", "Desert Cobblestone Wall", {"default_desert_cobble.png"},
```

Comment:
These changes are now no longer required.

Reply:
Agreed, not required but might be useful as an example.